src/cache/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface IRedisCacheOptions {
```

Comment:
maybe this interface can be a type that Picks from redis options and allows other properties with Partial as well
(for example, if someone wants to use redis sentinel, instead of host and port)

Reply:
i agree. but since ioredis is a peer dependency it is not possible